src/controllers/components/student-panel.js

```diff
@@ -32,4 +32,21 @@ export default class StudentPanelController {
         });
     }
 
+    static reserveItem(itemAddress) {
+        return saveItem(itemAddress).then(() => {
```

Comment:
I expected a name conflict if I tried calling the API-client's `saveItem` within a different function named `saveItem`. It seems to work, though, so I'll change this.